### ISSUE TITLE
summary(shc_result) reports "euclidean" when specifying custom matmet.

### ISSUE THREAD

**Issue by mebbert** [pkimes/sigclust2#6]
Hi,
I noticed that `summary(shc_result)` reports `dissimilarity = euclidean` when I specify a custom `matmet`. My understanding is that the `matmet`, `vatmet`, and `metric` are mutually exclusive. 

I assume this is just a bug in `summary.shc` and doesn't affect the actual results.

Here are my calls and result:

```
> shc_result <- shc(t(sig.heat), matmet = function(x){as.dist(1-cor(t(x), method = sample.dist))}, linkage="complete", null_alg = "2means", ci = "2CI")
> summary(shc_result)

shc object created using shc(..)
--------------------------------
Clustering Parameters:
    dissimilarity = euclidean
    linkage = complete
Testing Parameters:
    n_sim = 100
    icovest = 1
    ci = 2CI
    null_alg = 2means
    n_min = 10
    FWER control = FALSE
```
Really appreciate your help. Let me know if you need additional information.

**Comment by pkimes** [comment]
@mebbert, thanks for catching this!

You're right - this is just a bug in the summary output.
It should be fixed with the update I made in commit ce27658 (and therefore, the current release).

**Comment by pkimes** [comment]
Closing the issue as resolved.
Feel free to re-open if the issue still persists.
